ui-tests/src/test/java/io/syndesis/qe/steps/apiprovider/ApiProviderSteps.java

```diff
@@ -94,25 +112,29 @@ public void verifyThereAreWarnings(int num) {
         assertEquals("Wrong number of warnings", num, actual);
     }
 
-    @Then("^check operation \"([^\"]*)\" implementing \"([^\"]*)\" with status \"([^\"]*)\"$")
-    public void checkOperationImplementingWithStatus(String operationName, String operationPath, String operationStatus) {
+    @Then("^check operation \"([^\"]*)\" implementing \"([^\"]*)\" to \"([^\"]*)\" with status \"([^\"]*)\"$")
```

Comment:
Can you reword all gherkin steps here so it is clear that the step is connected with api-provider?